### PR TITLE
Belizairebr25.md

### PR DESCRIPTION
**Pull request by belizairebr25**
## Describe your project in 1-3 sentences
I will be building a glasses-computer utilizing the tapping of the thumb to different fingers as a keyboard. This will use  transparent OLED technology to display information from a raspberry pi. This is my first time working with displays, transistors and keyboard technology; I hope all goes well!

## Hack Club Slack handle
Brician Belizaire

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! **we're back on tuesday and will start reviewing PRs in the order that they came in, thanks for your patience :)**

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts & ready software

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: can you please format your document similar to the example in the [README](https://github.com/hackclub/winter#readme)? thank you!)_

our Jan 15 deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

**Review by exu3 — CHANGES_REQUESTED** [review]
Hey @belizairebr25! It looks like you edited the wrong file. Did you follow the instructions in the README and create your own file with your project idea? If you're unsure about anything, feel free to ask in #hardware-party on Slack—we're here to help!

**Comment by belizairebr25** [comment]
Dear Ella,
I believe I have resolved the issue(s), sorry for the confusion.
Thank You
Brician Belizaire

File: belizairebr25.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Why:** I give a lot of pre-written talks/speeches and it would be nice to have a teleprompter there to guide me if I forget where I am. It has also always been a dream of mine to make something like this and it would be awesome if that could become a reality.
+
+## Plan
+
+1. Design wiring schematics and paper drafts to guide when wiring and to find any potential issues with the idea, as well as to aid in knowing quantities of what is needed.
```
Comment (exu3) [inline belizairebr25.md:28]: What software might you be using to design this?

File: belizairebr25.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+2. Gather materials to construct the display and keyboard
+3. begin making keyboard prototypes with transistors, resistors, and arduino.
+4. Assemble the display and print the glasses frame.
```
Comment (exu3) [inline belizairebr25.md:31]: Will you be creating your own 3D model for the frame, or using an open source model? Could you clarify this in your project proposal?

File: belizairebr25.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| Raspberry Pi Zero W                                                                                                                            | https://www.amazon.com/gp/aw/d/B06XFZC3BX/ref=ox_sc_act_image_7?smid=AN9XFB4R34UOI&psc=1                                      | $62.90  |
+| Portable USB charger                                                                                                                           | https://www.amazon.com/gp/aw/d/B09H4GLZXT/ref=ox_sc_act_image_6?smid=A3CUYIPLALR9UA&psc=1                                     | $34.90  |
+| LCD display module                                                                                                                             | https://www.amazon.com/gp/aw/d/B082GFTZQD/ref=ox_sc_act_image_5?smid=A3B0XDFTVR980O&psc=1                                     | $14.75  |
+| Copper sheet roll for making keyboard contacts                                                                                                 | https://www.amazon.com/gp/aw/d/B0B21S6J5R/ref=ox_sc_act_image_4?smid=A3E294TQDZJYZC&psc=1                                     | $13.59  |
```
Comment (exu3) [inline belizairebr25.md:45]: Do you have tools to cut the copper sheet?

**Review by exu3 — CHANGES_REQUESTED** [review]
Neat idea, @belizairebr25. I have formatted your file using [Markdown](https://www.markdownguide.org/cheat-sheet/); I just copy/pasted what you had put originally, so please edit/correct if there is anything missing. 

Could you add some more details in your proposal (see comments below)? It's almost there!

**Comment by belizairebr25** [comment]
I believe I have resolved the specified issues.

**Review by exu3 — APPROVED** [review]
Great, thanks @belizairebr25! Can't wait to see you build this.

@hackclub/bank-ops will review and follow up with next steps via Slack. Happy hacking.

**Review by MelSmith104 — COMMENTED** [review]
hey @belizairebr25 this is super cool! i love this idea and it sounds like it'll be so much fun to build and test out! im dm'ing you on slack right now to chat about next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**